<launch>
    <arg name="delivery" default="true"/>
    <arg name="simulation" default="true"/>
    <arg name="return_home" default="false"/>
    <arg name="desired_velocity" default="0.7"/>
    <arg name="hover_time" default="5.0"/>
    <arg name="unpack_time" default="5.0"/>
    <arg name="z_delivery" default="0.5"/>
  
    <node name="offboard_node" pkg="offboard" type="offboard_node" output="screen">
        <param name="delivery_mode_enable" type="bool" value="$(arg delivery)"/>
        <param name="simulation_mode_enable" type="bool" value="$(arg simulation)"/>
        <param name="return_home_mode_enable" type="bool" value="$(arg return_home)"/>
        
        <param name="number_of_target" type="int" value="5"/>
        <param name="target_error" type="double" value="0.1"/>
        <param name="goal_error" type="double" value="0.2"/>
        <param name="z_takeoff" type="double" value="5.0"/>
        <param name="z_delivery" type="double" value="$(arg z_delivery)"/>
        <param name="land_error" type="double" value="0.1"/>
        <param name="takeoff_hover_time" type="double" value="5.0"/>
        <param name="hover_time" type="double" value="$(arg hover_time)"/>
        <param name="unpack_time" type="double" value="$(arg unpack_time)"/>
        <param name="desired_velocity" type="double" value="$(arg desired_velocity)"/>
        <param name="land_velocity" type="double" value="0.7"/>
        <param name="return_velcity" type="double" value="0.7"/>

        <param name="yaw_rate" type="double" value="0.05"/>
        <param name="yaw_error" type="double" value="0.03"/>

    </node>
</launch>
